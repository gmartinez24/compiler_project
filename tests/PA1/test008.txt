// Aum Palande
// Testing ident names and whitespace

main


int variable_one_1 = == 10;

int var_24_$ = - 12;

bool x_y_z ==== 3;