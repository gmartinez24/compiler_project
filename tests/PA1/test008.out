MAIN
INT
IDENT   variable_one_1
ASSIGN
EQUAL_TO
INT_VAL 10
SEMICOLON
INT
IDENT   var_24_
ERROR
ASSIGN
SUB
INT_VAL 12
SEMICOLON
BOOL
IDENT   x_y_z
EQUAL_TO
EQUAL_TO
INT_VAL 3
SEMICOLON
EOF